app/views/layouts/application.html.erb

```diff
@@ -6,16 +6,17 @@
 </head>
 <body>
   <div id="header">
-    <% if signed_in? -%>
+    <% if signed_in? %>
+      <%= link_to t(".new_recipe"), new_user_recipe_path(current_user) %>
       <%= button_to t(".sign_out"), sign_out_path, method: :delete %>
-    <% else -%>
+    <% else %>
       <%= link_to t(".sign_in"), sign_in_path %>
-    <% end -%>
+    <% end %>
   </div>
 
   <div id="flash">
-    <% flash.each do |key, value| -%>
-      <div id="flash_<%= key %>"><%=h value %></div>
+    <% flash.each do |key, value| %>
+      <div id="flash<%= key %>"><%=h value %></div>
```

Comment:
I'm curious why you dropped the `_` here?

Reply:
I was unclear why it was there to begin with, but looking at it again, it will make it easier to organize in the styling for flashes (flash_success, etc), so on second thought I think I will add back in.